setup.py

```diff
@@ -34,10 +35,28 @@ def __init__(self, attrs=None):
         self.ext_modules = [Extension('h3c', [])]
 
 
+try:
+    import pypandoc
+    long_description = pypandoc.convert('README.md', 'rst')
+    blacklist = [
+        'Build Status', 'Latest Version', 'Documentation Status',
+        'travis-ci.org'
+    ]
+    long_description = '\n'.join([
+        line for line in long_description.split('\n')
+        if not any(patt in line for patt in blacklist)
+    ])
+except Exception as e:
+    sys.stderr.write('Failed to convert README.md to rst:\n  {}\n'.format(e))
+    sys.stderr.flush()
+    long_description = open('README.md').read()
+
 setup(
     name='h3',
     version=h3_version,
-    description='Python bindings for H3, a hierarchical hexagonal geospatial indexing system developed by Uber Technologies',
+    description=
+    'Python bindings for H3, a hierarchical hexagonal geospatial indexing system developed by Uber Technologies',
+    long_description=long_description,
     author='Uber Technologies',
     author_email='David Ellis <d.f.ellis@ieee.org>',
```

Comment:
It was lots of fun learning how to make bindings in so many different languages. :)

I still think JNR-FFI is better designed than JNI in Java, but I understand why @isaacbrodsky replaced it.

Reply:
JNI vs. JNR-FFI reminds me of the whole Python module/Cython vs. Python ctypes. seems to be the former is more user friendly than the latter in both cases.

Reply:
JNI more user friendly than JNR-FFI? I think you've got that backwards, JNI is madness, and JNR-FFI is far clearer code.

Reply:
Thanks for the review guys!

Reply:
BTW I have no opinions on JNI vs JNR lol...

Reply:
@dfellis if you have no C code in the library, because you use pure Java with JNR-FFI, then you need the user to install h3 independently of your library. From the perspective of Java, the C library is completely external. This is the current issue with Python too. Only by using the native module interface in both language can you include the C code in the language library.